README.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+
+```{r, include = FALSE}
+#| label: setup
+#| include: false
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "75%",
+  warning = FALSE,
+  message = FALSE,
+  fig.retina = 2,
+  fig.align = 'center'
+)
+library(tidyverse)
+```
+
+# srvyexploR
+
+The **srvyexploR** package provides datasets used in the book "Exploring Complex Survey Data Analysis in R: A Tidy Introduction with srvyr". This will help readers follow along with examples and work on exercises.
```

Comment:
```suggestion
The **srvyexploR** package provides datasets used in the book "Exploring Complex Survey Data Analysis in R: A Tidy Introduction with srvyr". This will help readers follow along with the examples and work through the exercises.
```